spec/fabricators/fabricator.rb

```diff
@@ -4,7 +4,7 @@
   title { Faker::HarryPotter.book }
   description { Faker::HarryPotter.quote }
   price { Faker:: Number.decimal(2) }
-  category  
+  category  { Fabricate(:category) }
```

Comment:
sweet; good work on fabricator lets use it for the parts that we have in our tests